daemon/src/wallet.rs

```diff
@@ -95,12 +99,20 @@ impl Actor {
             ElectrumBlockchain::from(client),
         )?;
 
+        // UTXOs chosen after coin selection will only be locked for a
+        // few wallet sync intervals. UTXOs which were actually
+        // included in published transactions should be marked as
+        // spent by the internal bdk wallet by then. UTXOs which ended
+        // up not being used are expected to be safe to be reused by
+        // then without incurring in double spend attempts.
+        let time_to_lock = SYNC_INTERVAL * 4;
```

Comment:
@thomaseizinger: From what we previously discussed. I removed all logic based on Bitcoin networks accordingly.